test/new-e2e/system-probe/test-json-review/main.go

```diff
@@ -9,30 +9,76 @@ import (
 	"bufio"
 	"encoding/json"
 	"fmt"
+	"io/fs"
 	"log"
 	"os"
+	"path/filepath"
+	"regexp"
+	"sort"
 	"strings"
 	"time"
 
 	"github.com/fatih/color"
 )
 
+var CIVisibility = filepath.Join("/", "ci-visibility")
```

Comment:
nit: I'm not sure `filepath.Join` is helping here, because you are already hardcoding a separator of `/`. I think you can reasonably just use `"/ci-visibility"`